### DISCUSSION TITLE
Raspberry Pi installation

### DISCUSSION THREAD

**Discussion by kevin-co-opts** [coqui-ai/STT#2016] [Q&A]
Excuse the newbie question but when I try and install STT on a raspberry pi 4 I keep getting wheel errors
`ERROR: Could not find a version that satisfies the requirement stt (from versions: none)`
`ERROR: No matching distribution found for stt`
I've tried several different approaches used the latest Raspbian Lite and Ubuntu  server 20.04 but I keep getting the same error. Clearly it's something I have not done, any ideas?

**Comment by SenorGato** [comment]
Am encountering the same error in archlinux armv7.

**Reply by kevin-co-opts** [reply]
I'm rebuilding the binaries as it seems its the only solution. Also Gitter is a better place to get help

**Comment by reuben** [comment]
Have you tried to install our pre-built wheels for RPi from our [release page](https://github.com/coqui-ai/STT/releases/v1.2.0)?

**Reply by kevin-co-opts** [reply]
Trying that as we speak, I'll let you know how it goes
